modules/manage/partials/audit-logging.adoc

```diff
@@ -16,6 +16,8 @@ With audit logging enabled, there should be no noticeable changes in performance
 ifndef::env-cloud[]
 NOTE: Audit logging is configured at the cluster level. Redpanda supports excluding specific topics or principals from auditing to help reduce noise in the log. Audit logging is disabled by default.
 
+*Redpanda Console*: To view audit events in Redpanda Console, set `console.auditLog.enabled: true`. See xref:console:config/configure-console.adoc[].
```

Comment:
No viewing Redpanda Core audit logs in Redpanda Console is not the feature here. It's only about emitting audit / access logs for Console itself on the console.